stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDocumentProvider.java

```diff
@@ -138,11 +141,16 @@ public NodeDescriptor getNodeDescriptor(Object element) {
   public void highlightElement(Object element, int color) {
     verifyThreadAccess();
 
-    View highlightingView = getHighlightingView(element);
-    if (highlightingView == null) {
+    HighlightableDescriptor highlightableDescriptor = getHighlightableDescriptor(element);
+    if (highlightableDescriptor == null) {
       mHighlighter.clearHighlight();
     } else {
-      mHighlighter.setHighlightedView(highlightingView, color);
+      mHighlightBoundsRect.setEmpty();
+      highlightableDescriptor.getBoundsForHighlighting(element, mHighlightBoundsRect);
```

Comment:
I'm worried there's a bug hazard here. It looks like we're asking the descriptor for the highlighting bounds for an element, but not really specifying what that's relative to. It's apparent later, when we call `getViewForHighlighting`, what those bounds should be relative to. But these calls are disjoint; the protocol seems confusing or clumsy for the implementor.

Can we merge these methods? e.g. `@Nullable View getViewAndBoundsForHighlighting(Object element, Rect bounds)` ... so basically rename `getViewForHighlighting` and add the `Rect` parameter.